### PR TITLE
Adding vim, VS Code and some tmux shortcuts

### PR DESCRIPTION
**Pull request by ratnadeepb**
Adding a small guide to vim along with a config file and added a few tmux options

### CODE REVIEW

**Comment by ratnadeepb** [comment]
Adding VS Code startup README

File: README.md [hunk 1]
```diff
@@ -8,3 +8,6 @@ Please do pull requests for additional tools.
 
 If you want to use a tool, copy the dotfile(s) for it to your home directory (unless otherwise noted).
 The `README.md` in each tool's directory should include an overview of the tool, an overview of how to use it (for beginners), and a set of links for people to learn more.
+
+## Regarding VIM config
```
Comment (gparmer) [inline README.md]: Please remove this.

File: tmux/README.md [hunk 2]
```diff
@@ -42,3 +42,16 @@ The next keypresses you make will be interpreted as a command to `tmux`, which a
 - [source and documentation](https://github.com/tmux/tmux)
 - [FAQ](https://github.com/tmux/tmux/wiki/FAQ)
 - [man page](http://man.openbsd.org/OpenBSD-current/man1/tmux.1)
+
+## Some tmux shortcuts
+- Open a new session
+    - tmux new
+    - tmux new dev -> named session called "dev"
+- Tmux command mode (Ctrl + b)
```
Comment (gparmer) [inline tmux/README.md]: Doesn't this conflict with the actual config that redefines the control code?

File: VS_Code/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+# Visual Studio Code
```
Comment (gparmer) [inline VS_Code/README.md]: Sorry: I just merge @bushidocodes work on vscode. You can add/modify that.

Sorry I'm causing you issues here!

File: vim/README.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
+# VIM
```
Comment (gparmer) [inline vim/README.md:1]: This all looks great. I want to merge the vim stuff as-is.

File: tmux/README.md (hunk 2 above)
Comment (ratnadeepb) [reply tmux/README.md]: Could you clarify this point please. I have so far had no issues with named sessions.

File: VS_Code/README.md (hunk 3 above)
Comment (ratnadeepb) [reply VS_Code/README.md]: No worries. That was a brain fade on my part. I will make the necessary changes.

File: README.md (hunk 1 above)
Comment (ratnadeepb) [reply README.md]: Done in commit e0f7e91

File: VS_Code/README.md (hunk 3 above)
Comment (ratnadeepb) [reply VS_Code/README.md]: In commit bd616f5

**Comment by ratnadeepb** [comment]
Incorporated all suggestions

File: tmux/README.md (hunk 2 above)
Comment (gparmer) [reply tmux/README.md]: The tmux config that is in the tree uses a different control sequence, not `cntl + b`, so including documentation that implies it is `cntl + b` is not quite right.

Comment (ratnadeepb) [reply tmux/README.md]: In the tmux config I use `cntl+a` is the secondary `<prefix>` but `cntl+b` still works.

Comment (ratnadeepb) [reply tmux/README.md]: I changed it to `cntl + t`. I had assumed, erroneously, that since unbind was not used `cntl + b` would still work.

Comment (ratnadeepb) [reply tmux/README.md]: Commit e288d5c

**Pull request merged**